test-plugins/org.yakindu.sct.generator.java.test/test-gen/org/yakindu/sct/generator/java/test/TimedTransitions.java

```diff
@@ -26,24 +26,32 @@
 	private TimedTransitionsStatemachine statemachine;	
 	private VirtualTimer timer;
 	
+	
+	
 	@Before
-	public void setUp() {
+	public void timedTransitions_setUp() {
 		statemachine = new TimedTransitionsStatemachine();
 		timer = new VirtualTimer();
 		statemachine.setTimer(timer);
 		statemachine.init();
 	}
 
 	@After
-	public void tearDown() {
+	public void timedTransitions_tearDown() {
 		statemachine = null;
+		timer = null;
 	}
 	
 	@Test
 	public void timer01() {
 		statemachine.enter();
 		assertTrue(statemachine.isStateActive(State.main_region_Start));
-		timer.timeLeap(2030);
+		for(int i = 1; i <= 10; i++) {
+			timer.timeLeap(200);
+			statemachine.runCycle();
+		}
+		timer.timeLeap(30);
+		statemachine.runCycle();
```

Comment:
This is not right to trigger a runcycle after timeleap that is not a multiple of the cycle period